commander3/parameter_files/param_BP10.0.txt

```diff
@@ -0,0 +1,389 @@
+**************************************************************
```

Comment:
Can we name this file BP 9.x instead because there is already a BP10 run that is on the adc branch and this will conflict with that name

Reply:
Shure, thats ok by me, name it BP9.1 or something.

Reply:
I just renamed it

Reply:
Great!